packages/redux/src/orders/selectors.ts

```diff
@@ -885,91 +859,175 @@ const denormalizeOrder = (
           )
           .filter(Boolean) as OrderItemEntityDenormalized[])
       : undefined,
-    returnOptions: order.returnOptions
-      ? (order.returnOptions
-          .map(returnOptionId =>
-            denormalizeReturnOption(returnOptionId, returnOptions, merchants),
-          )
-          .filter(Boolean) as ReturnOptionEntityDenormalized[])
-      : undefined,
-    returns: order.returns
-      ? (order.returns
-          .map(returnId => denormalizeReturn(returnId, returns))
-          .filter(Boolean) as ReturnEntity[])
-      : undefined,
   };
 };
 
 /**
- * Get orders result, denormalized.
+ * Get orders result. This returns order summaries denormalized.
+ * Organized by order summary.
+ */
+export const getOrderSummariesResult: (
+  state: StoreState,
+  query?: GetUserOrdersQuery,
+) => OrderEntityDenormalized[] | OrderSummariesDenormalized | undefined =
```

Comment:
This return type does not match the output of the function anymore.

Reply:
Indeed, fixed it, removed the `orderEntityDenormalized[]` 👍